### PR TITLE
Ht 2

### PR DESCRIPTION
**Pull request by sancho-sumy**

### CODE REVIEW

File: src/components/Courses/components/SearchBar/SearchBar.jsx [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 import styles from './SearchBar.module.css';
 
 export function SearchBar({ onSubmit, onChange }) {
 	return (
-		<form className={styles.container} onSubmit={onSubmit}>
+		<Form id='search-form' className={styles.container}>
 			<Input
 				placeholderText='Enter course name...'
-				name='search'
+				name='q'
```
Comment (obriukhanov) [inline src/components/Courses/components/SearchBar/SearchBar.jsx:12]: ?

**Review by obriukhanov — CHANGES_REQUESTED** [review]
Logged In User still able to go to `/login & /register` routes.
Improve routing.

File: src/components/Courses/components/SearchBar/SearchBar.jsx (hunk 1 above)
Comment (sancho-sumy) [reply src/components/Courses/components/SearchBar/SearchBar.jsx:12]: This is a search mechanism that uses query parameters and action/loader from react-router. Inspired by this tutorail: https://reactrouter.com/en/main/start/tutorial#get-submissions-with-client-side-routing.

**Comment by sancho-sumy** [comment]
> Logged In User still able to go to `/login & /register` routes. Improve routing.

Fixed.

**Pull request merged**